paths/graphql/03-first-query.md

```diff
@@ -0,0 +1,66 @@
+
+ 1. Paste this query into the [GraphQL explorer](https://developer.github.com/v4/explorer/).
+
+         query RecentPRs {
+           viewer {
+             name
+             pullRequests(last: 5) {
+               edges {
+                 node {
+                   title
+                   body
+                   url
+                 }
+               }
+             }
+           }
+         }
+
+    > To identify what the different parts of this query are doing, click "Tell me why" below. We could easily change this query to ask for different information. If you'd like, feel free to do that in your own query!
+
+  1. Click the "Execute Query" button (it looks like a Play button) to run the query that is in the left pane and display the results in the right pane.
+
+  This example shows that with the same code, we can get updated data! Any time you have to manually search for information, try to find out if and how you could use an API to do it for you.
+
+tell-me-why: |
+
+  ### What is a query?
+
+  A query is an opportunity to ask an existing service, say, GitHub to provide specific information based on the context of your question. Our last "username and avatar exercise" would be similar to asking someone their name and asking for a picture, except with a **query** we can ask GitHub to get the information from _hundreds_ (or _thousands_, or even _millions_) users, repositories, issues, commits, and pull requests.
+
+  Additionally, you can use the response to your **query** (or question) to dynamically update and potentially alter the way your content looks.
+
+  During the introduction to the course, we identified that using queries and API would allow you to automatically get consistently updated information and use it to power your application or website. As an example, if you wanted to list the latest commits made across GitHub manually, you would need to do a lot of searching and manually update your application. More importantly, in the time it took your to make your changes, the data might be obsolete. Using queries and APIs, we can automatically gather the latest data, display it, and schedule when that data needs to be updated.
+
+  ### Our query
+
+  - [**viewer:**](https://developer.github.com/v4/guides/using-the-explorer/) Who is currently viewing this information (you!)
+  - [**name:**](https://developer.github.com/v4/reference/object/user/#fields) The name of the viewer.
+  - [**pullRequests(last: #):**](https://developer.github.com/v4/reference/object/user/#fields) The last (most recent) pull requests that are relevant to that viewer. The `#` sign can be used to limit the number of pull requests you want returned.
+  - [**edges:**](https://developer.github.com/v4/reference/object/pullrequestedge/) An edge in a connection.
+  - [**node:**](https://developer.github.com/v4/reference/object/pullrequest/) The item at the end of an edge.
```

Comment:
A "node" is "the item at the end of an edge" and an "edge" is an "edge"? I would :heart: to see us take a step back and explain these terms in ways that make them more accessible for someone with no exposure to GraphQL.